encoding/geojson/geojson_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
 
 	"github.com/d4l3k/messagediff"
+	"github.com/stretchr/testify/require"
```

Comment:
This brings in `testify` but please keep it - we'll move to `testify` eventually (#126 🙄 ).

Reply:
(and `testify` is already in `go.mod` due to the PostGIS example)

Reply:
haha, i'll see if i can give you a hand :)
i got fed up with hard to read differences